dist/40-10007-schema.json

```diff
@@ -269,10 +269,6 @@
             "III",
             "IV"
           ]
-        },
-        "maiden": {
```

Comment:
Should this be somewhere else in the schema?

Reply:
Good idea @jbalboni. Maiden has been relocated to applicant name.
Also, in looking at how other forms handle this, I noticed that there is an outstanding task for determining the validation for mothersMaidenName in the HCA schema. I could not find a ticket for this, should one be created?